src/pages/debrief/cat-cpc/components/debrief-cpc-card/debrief-cpc-card.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Question, Question5 } from '@dvsa/mes-test-schema/categories/CPC';
+
+export type QuestionUnion = Question
+| Question5;
+
+@Component({
+  selector: 'debrief-cpc-card',
+  templateUrl: 'debrief-cpc-card.html',
+})
+export class DebriefCPCCardComponent implements OnInit {
+
+  @Input()
+  public question1: Question;
+
+  @Input()
+  public question2: Question;
+
+  @Input()
+  public question3: Question;
+
+  @Input()
+  public question4: Question;
+
+  @Input()
+  public question5: Question5;
+
+  @Input()
+  public overallScore: number;
+
+  @Input()
```

Comment:
dont think this needs the @Input() decorator as its not passed from the Debrief page

Reply:
kept this as its now being passed from Debrief page